src/nvim/tui/tui.c

```diff
@@ -1437,6 +1449,10 @@ static void patch_terminfo_bugs(TUIData *data, const char *term,
   bool mate_pretending_xterm = xterm && colorterm
     && strstr(colorterm, "mate-terminal");
   bool true_xterm = xterm && !!xterm_version;
+  bool cygwin = terminfo_is_term_family(term, "cygwin");
+#ifdef WIN32
+  bool conemu_ansi = uv_guess_tty(data->out_fd) & UV_TTY_CONEMU;
```

Comment:
Default to false, then we can avoid some ifdefs below (which is nice for extra coverage by coverity etc.)